index.html

```diff
@@ -3,184 +3,196 @@
-            <img src="https://assets.csh.rit.edu/pubsite/science_house_2019.jpeg" alt="A sign that reads Science House with CSHers in the background. Taken at Fall Camping 2019" />
-        </picture>
+    <div class="banner-overlay-div"></div>
+    <div class="banner-text-div">
+    <div class="banner-text">
+        <p class="banner-h1-text">Welcome to Computer Science House</p>
+        <p class="banner-btn"><a class="banner-btn-item" href="https://csh.rit.edu">Learn more &#8594;</a></p>
     </div>
-    <div>
-        <picture>
-            <source srcset="https://assets.csh.rit.edu/pubsite/holiday_dinner_2019.webp" type="image/webp" />
-            <source srcset="https://assets.csh.rit.edu/pubsite/holiday_dinner_2019.webp" type="image/jpeg" />
-            <img src="https://assets.csh.rit.edu/pubsite/holiday_dinner_2019.webp" alt="CSHers pose for a group photo after Holiday Dinner 2019" />
-        </picture>
     </div>
-    <div>
-        <picture>
-            <source srcset="https://assets.csh.rit.edu/pubsite/fall_camping_group_2019.webp" type="image/webp" />
-            <source srcset="https://assets.csh.rit.edu/pubsite/fall_camping_group_2019.webp" type="image/jpeg" />
-            <img src="https://assets.csh.rit.edu/pubsite/fall_camping_group_2019.webp" alt="A group of CSHers at Fall Camping 2019" />
-        </picture>
-    </div>
-</div>
-<div class="gray-wrapper">
-    <div class="container spaced" id="homepage-about">
-        <div class="row">
-            <div class="col-12 col-lg-6">
-                <picture>
-                    <source srcset="https://assets.csh.rit.edu/pubsite/about_image.webp" type="image/webp" />
-                    <source srcset="https://assets.csh.rit.edu/pubsite/about_image.jpeg" type="image/jpeg" />
-                    <img class="rounded" src="https://assets.csh.rit.edu/pubsite/about_image.jpeg" alt="A group of CSHers at our annual camping trip">
-                </picture>
+</section>        
+
+
+<h2 style="text-align: center; font-size: 1.4em;">Getting more done <span style="color: #b0197e;">after 2am</span> than most people do all day.</h2>
+
+<!-- Triple image set -->
+<div class="width_capped">
+    <div class="container">
+        <div id="triple_image_wrapper">
+        <article class="card" role="article">
+            <a class="triple_a" href="/about/">
+            <div class="card-text">
+                <h2 class="card-title">About us</h2>
             </div>
-            <div class="col-12 col-lg-6">
-                <p>Since 1976, <strong>Computer Science House</strong> has provided a revolutionary living and learning environment for its members. With unique facilities, an emphasis on hands-on learning, and a strong social atmosphere, CSH helps its members grow as professionals and more.</p>
-                <a class="btn btn-primary" href="/about/">More about CSH</a>
+            <img class="card-image" src="assets/images/disk.png" />
```

Comment:
Maybe fuzz the names on that image?

Reply:
I don't think that would look good. If you think it's a bad idea to have that image up at all I'll just replace it entirely